README.md

```diff
@@ -5,129 +5,162 @@
  - `ClusterEnv` is essentially a wrapper around the node socket. The socket belongs to one of the nodes.
+ - `action :: ClusterEnv -> IO a` is a user action which has access to a `cardano-node` in a cluster via `Cardano.Api`.
 
 Use
-```
+```haskell
 withFundedCluster :: PlutipConfig -> [[Lovelace]] -> (ClusterEnv -> [KeyPair] -> IO a) -> IO a
 ```
-to additionaly receive keys prefunded with specified fund distributions (i.e. Key 1 with [1 Lovelace] and Key 2 with [2 Lovelace, 4 Lovelace]).
+to additionaly receive keys prefunded with specified fund distributions (e.g. Key 1 with `[1 Lovelace]` and Key 2 with `[2 Lovelace, 4 Lovelace]`).
 
-Additionaly there's helpers `startCluster`, `startFundedCluster`, `stopCluster` useful when you want your cluster to keep running, instead of close after the IO action is completed.
+Additionaly there are helpers `startCluster`, `startFundedCluster`, `stopCluster` which are useful when you want your cluster to keep running, instead of shutting down after the IO action is completed.
 
-Full example: 
-```
+Example:
+```haskell
 import Data.Default (Default (def))
 import Plutip.CardanoApi (currentBlock)
 import Plutip.Cluster (withFundedCluster)
 import Plutip.Keys (cardanoMainnetAddress)
 
 main = withFundedCluster def [[ada 1], [ada 2, ada 4]] $ \cenv [key1, key2] -> do
   -- You have a default cluster using a temporary directory for storage and 7 Ada to use wisely.
-  -- You can i.e. query the node for block number with 
+  -- You can query the node for block number with
   res <- currentBlock cenv
   ...
   -- See Plutip.CardanoApi for example queries and construct your own queries with Cardano.Api.
   -- To make use of your keys, also use Cardano.Api.
 
 ada = (*) 1_000_000
-
 ```
 
 ### As an executable
 
-Plutip provides a `local-cluster` executable. You can build it and run with nix package manager:
-```
+Plutip provides a `local-cluster` executable.
+You can build it and run with Nix:
+```bash
 nix run github:mlabs-haskell/plutip#plutip-core:exe:local-cluster -- --help
 ```
-Available options mostly match the `withFundedCluster` interface, see `--help` and [README](local-cluster/README.md).
 
-### Via CTL
+Available options mostly match the `withFundedCluster` interface, see `--help` and [local-cluster README](local-cluster/README.md) for detailed description of the arguments.
+
+### Via CTL for contract testing
 
-CTL is a purescript sdk for creating dapps. One of its features is ability to test contracts with private networks, see [plutip-testing](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/plutip-testing.md).
+[CTL](https://github.com/Plutonomicon/cardano-transaction-lib) is a PureScript SDK for creating DApps.
+One of its features is the ability to test contracts on disposable private networks which Plutip sets up, see [plutip-testing](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/plutip-testing.md).
```

Comment:
Please mention that users should not install plutip if the want to use it with ctl, because it provides everything needed